<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="DESKTOP-5SRPD8A" failures="1" tests="8" name="Personal Loan" time="0.063" errors="0" timestamp="2022-04-06T20:18:50 SGT">
  <testcase classname="testng_tutorial.Day3" name="apiLogin" time="0.005"/>
  <testcase classname="testng_tutorial.Day2" name="print" time="0.003"/>
  <testcase classname="testng_tutorial.Day3" name="webLogin" time="0.001"/>
  <testcase classname="testng_tutorial.Day3" name="mobileLogin" time="0.002"/>
  <testcase classname="testng_tutorial.Day3" name="mobileLogin2" time="0.0"/>
  <testcase classname="testng_tutorial.Day3" name="mobileLogin3" time="0.001"/>
  <testcase classname="testng_tutorial.Day3" name="sampleListener" time="0.004">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at testng_tutorial.Day3.sampleListener(Day3.java:23)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- sampleListener -->
  <testcase classname="testng_tutorial.Day1" name="demo2" time="0.001"/>
  <testcase classname="testng_tutorial.Day1" name="demo" time="0.0">
    <ignored/>
  </testcase> <!-- demo -->
</testsuite> <!-- Personal Loan -->
